iceoryx_hoofs/include/iceoryx_hoofs/internal/cxx/optional.inl

```diff
@@ -90,6 +94,7 @@ inline optional<T>::optional(in_place_t, Targs&&... args) noexcept
     construct_value(std::forward<Targs>(args)...);
 }
 
+// AXIVION Next Construct AutosarC++19_03-A13.3.1 : It's safe and constrained by SFINAE
```

Comment:
Nitpick: In general, the question "Why is it safe?" needs to be answered. It's not clear which SFINAE type trait you are talking about, maybe add something like `see other operator=(T&& )`